workers/unity/Packages/io.improbable.gdk.core/Worker/ConnectionHandlers/MockConnectionHandler.cs

```diff
@@ -211,6 +290,29 @@ public RemoveEntityTemplateDynamicHandler(EntityTemplate template, long entityId
             }
         }
 
+        private class MockOutgoingCommandHandler : IOutgoingCommandHandler
+        {
+            private long nextRequestId = 1;
+
+            private readonly Action<long, Type, ICommandRequest> onSendCommand;
+
+            internal MockOutgoingCommandHandler(Action<long, Type, ICommandRequest> onSendCommand)
+            {
+                this.onSendCommand = onSendCommand;
+            }
+
+            public CommandRequestId SendCommand<TRequest>(TRequest request, Entity sendingEntity = default) where TRequest : ICommandRequest
+            {
+                onSendCommand(nextRequestId, typeof(TRequest), request);
+                return new CommandRequestId(nextRequestId++);
+            }
+
+            public void SendResponse<T>(T response) where T : ICommandResponse
+            {
+                // Do nothing
```

Comment:
We should either implement this or have this throw a [`NotImplementedException`](https://docs.microsoft.com/en-us/dotnet/api/system.notimplementedexception?view=netcore-3.1)

Reply:
If I throw the exception, it would require tests to have try-catch when they call `CommandSystem.SendResponse`